### ISSUE TITLE
Add Arabic language for a fee 

### ISSUE THREAD

**Issue by mahdi-salah** [CorentinJ/Real-Time-Voice-Cloning#1059]
Hello my friend, 
I want to add the Arabic language to the program for a fee. If you agree, how can I contact you?

**Comment by TrycsPublic** [comment]
email us at admin@trycs.net we may be able to do it for a fee, we accept XMR only, no paypal or card.

**Comment by TrycsPublic** [comment]
> Hi my friend I have this topic for my thesis and I need send me your schedule when you can finalize it and how much the price for doing this program 
>Best Regards

Send the details, it will take approximately $1000 for a well fine-tuned model.
You’ll also need to have a decent dataset approx: 8000+ speakers for encoder and 3000+ for SYN/VOC

It could take from 2 weeks to 1 month depends how much you pay.

**Comment by TrycsPublic** [comment]
> hi my friend'
> He has dataset that I will send to you. I want you to do it for me and a video that converts the Arabic Text to Speech Conversion. After I watch the video, I will transfer the amount to you and you convert the code and work steps
> http://en.arabicspeechcorpus.com/
> You can download the data needed for your project in the above link..
> Best Regards

Training a model costs hardware and money, you can ask for a middleman with a decent reputation to make this deal. Or you can pre-pay 50%. 

choose a model:

low level model: $500
mid level : $1000
highly trained: $4000-$10000

**Comment by TrycsPublic** [comment]
You need to proof you are not folding a trained model, you need to have a middleman or pay in half first for me to start training a new model.

**Comment by TrycsPublic** [comment]
>Hi
>I'm reading this conversation:
>https://github.com/CorentinJ/Real-Time-Voice-Cloning/issues/1059
>
>Is it possible to clone someone's voice in Italian language?
>What would the cost be?
>I wait your reply, please
>Thanks 
>Shayla

depends:

low level model: $500
mid level : $1000
highly trained: $4000-$10000

**Comment by kushwaha-Khushi-0612** [comment]
> Hello my friend, I want to add the Arabic language to the program for a fee. If you can help, I will contact you.

Yes Can you be more brief on you idea, and how you want it to be done
